Reader/CsvReader.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+namespace A5sys\FecBundle\Reader;
+
+use A5sys\FecBundle\Exception\FecException;
+use Symfony\Component\HttpFoundation\File\File;
+
+/**
+ * Dumps a associative array in a file
+ */
+class CsvReader implements ReaderInterface
+{
+    const CHARSET = 'ISO-8859-15';
+    const DATEFORMAT = 'Ymd';
+    const FLOAT_NB_DECIMALS = 2;
+    const FLOAT_DECIMAL_SEPARATOR = ',';
+    const FLOAT_THOUSAND_SEPARATOR = '';
+
+    protected $separator = "\t";
+
+    /**
+     * Constructor
+     * @param string $separator
+     */
+    public function __construct($separator)
+    {
+        $this->separator = $separator;
+    }
+
+    /**
+     * Reads the CSV FEC
+     * @param File  $file
+     * @return array<array>
+     */
+    public function read(File $file)
+    {
+        $splFileObject = $file->openFile('r');
+
+        // write csv data
+        return $this->readCsv($splFileObject);
```

Comment:
I think it should be 

    $data = $this->readCsv($splFileObject);

and 

    return $data

at the end of the method